appserver/tests/appserv-tests/devtests/deployment/run_test.sh

```diff
@@ -169,5 +169,7 @@ case $OPT in
     list_test_ids;;
   run_test_id )
     run_test_id $TEST_ID ;;
+  deployment_all )
```

Comment:
Why we need to explicitly define deployment_all target. Isn't it taken care by run_test_id